configure.ac

```diff
@@ -73,6 +73,41 @@ dnl   automatically (see the automake manual's FAQ for justification).
 drivers="exdriv.lo pgdriv.lo cadriv.lo hgdriv.lo lxdriv.lo  vtdriv.lo hidriv.lo psdriv.lo ttdriv.lo cwdriv.lo gldriv.lo nedriv.lo hpdriv.lo lsdriv.lo nudriv.lo qmdriv.lo vadriv.lo cgdriv.lo nexsup.lo rvdriv.lo xmdriv.lo tkdriv.lo pkdriv.lo xadriv.lo"
 
 dnl   Not all platforms have X
+have_x11=no
+have_xt=no
+AS_IF([ test "x$no_x" = "x"],
+      [
```

Comment:
I think there's something awry with the logic here.  `AC_PATH_XTRA` is invoked above, and, from the fine `autoconf` manual, section 5.11:

> If [finding X fails], or the user gave the command line option '--without-x', set the shell variable 'no_x' to 'yes'; otherwise set it to the empty string.

The true state of this `AS_IF` conditional is when `test "x$no_x" = "x"` succeeds, which requires that `$no_x` is the empty string and, from above,  `X` has already been found, so evaluating the body of the conditional is extra work.

Conversely, if `X` has not been found, `$no_x` is set to `yes`, but unfortunately that's also the value when the user explicitly doesn't want `X`, signified by setting the `--without-x` command line parameter.

I think the sense of the conditional is backwards, but that it won't work anyway because of the degeneracy between `AC_PATH_EXTRA` failing and the user setting `--without-x`

Reply:
How about checking `$with_x` directly and use `pkg-config` as an additional search method to `AC_PATH_XTRA`?

Reply:
I have added a commit below that does what I suggested by using `$with_x`.